File: src/main/java/edu/dcccd/trans/repository/RoleRepository.java
package edu.dcccd.trans.repository;

import edu.dcccd.trans.entity.Role;
import org.springframework.data.repository.CrudRepository;
import org.springframework.data.repository.query.Param;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface RoleRepository extends CrudRepository<Role, Long> {
    List<Role> findByUserId( @Param("user_id") Long user_id );
}

File: src/main/kotlin/edu/dcccd/trans/repository/RoleRepository.kt
package edu.dcccd.trans.repository

import edu.dcccd.trans.entity.Role
import org.springframework.data.repository.CrudRepository
import org.springframework.data.repository.query.Param
import org.springframework.stereotype.Repository

@Repository
interface RoleRepository : CrudRepository<Role, Long> {
    fun findByUserId(@Param("user_id") user_id: Long?): List<Role>
}